cfme/containers/provider/__init__.py

```diff
@@ -270,9 +271,19 @@ def num_image_registry_ui(self):
         return int(self.get_detail("Relationships", "Image Registries"))
 
 
+class ProjectAllView(BaseLoggedInPage):
+
+    table = Table(locator="//div[@id='list_grid']//table")
+
+    @property
+    def is_displayed(self):
+        return match_page(summary='Project')
```

Comment:
Looks like copypasta from ProjectAllView class in project.py.

Add table.is_displayed verification to all of these since you don't have all of the other page elements widgetized.